.github/dependabot.yml

```diff
@@ -13,7 +13,7 @@ updates:
     rebase-strategy: "disabled"
     ignore:
       - dependency-name: "@types/node"
-        versions: ["17", "18"]
+        versions: [">=20"]
```

Comment:
Should this be `<=20` since won't this ignore any 20.x versions?

Reply:
Ah, yeah, that should be:
```suggestion
        versions: [">=21"]
```
to ignore 21+. Fat fingered that version 🤦